### ISSUE TITLE
Create Lists without  loosing Groups 

### ISSUE THREAD

**Issue by argyrism** [soonick/poMMo#8]
Hi guys i'm working on the old code from 2008 that is not messy and i managed to add lists to pommo, i did not finish it yet but i belive that the ability to have separate lists for every mailing is usefull.

**Comment by soonick** [comment]
Hello argyrism,

I did make a lot of changes to the code structure with the intention to make it simpler, I am sorry to hear you find it messy.

As I told you in my other message, If you need help understanding the code I will be happy to help you.

**Comment by argyrism** [comment]
Hi soonick, 
i appreciate your efforts to make all these improvements, if you upload a new zip file with the files needed without repeated files in other folders this will help us a lot, i'm near finishing the lists (make separate lists for every mailing) like groups but without rules , just put subscribers to a list...

**Comment by soonick** [comment]
Hello argyrism,

I didnt understand what you are asking me for. All the code and changes I have done are in the repository, and you can download a zip file in the download page. If you give me more details maybe I will be able to put together what you need.

Thank you

**Comment by argyrism** [comment]
Hi soonick,
i have downloaded the files from the repository, but inside the zip files there are other files that are not used by the script, some files that were used by the old version are there and they are a bit confusing if somebody wants to see all the files quickly, thats what i meant from the beginning as "messy" , the code is fine, everything good.

**Comment by soonick** [comment]
Hello argryrism,

I dont know which files you are talking about. I removed a lot of unused files from previous versions. If you donwload from my master repository you should get the files you need.

**Comment by enginenr9** [comment]
I'm confused, argyrism, did you ever finished the lists option?

**Comment by argyrism** [comment]
Hello , yes i have finished the lists option, they work in parallel , you have both the option to select group and list, but i did all on my own , i did not get pommo from github,i started from the old version and i implemented a few features , like bounce handling , inline attachments , lists , tracking , but the pommo version i created is not following this one here so the guys might need a lot of effort to get features i implemented, they have fixed many issues that i did not, if they are interested i can send my version , maybe something usefull will be implemented from both versions, i dont have the time to develop it any more, regards, Argyris

**Comment by enginenr9** [comment]
I'm also trying to figure out how to add a option to create users lists but have no idea where to start - my php knowledge is still very basic.
Can you to share your version?

**Comment by argyrism** [comment]
my version uses a different database and there are 3-4 db connection 
files that have to be edited, i find it easier to look back on how i did 
the lists option and tell you, if you are interested i can see it as 
soon as i have time and inform you

**Comment by enginenr9** [comment]
Appreciate your help then. Thank you

**Comment by argyrism** [comment]
my apologies , i do not have a lot of free time so i have prepared an early version of the modified script i have developed, you can try it , just make the database manually and configure and place the db files one level before public directory. 

http://www.moukouros.com/modified_pommo.rar
and
http://www.moukouros.com/pommo_db.rar 
thanks!

**Comment by enginenr9** [comment]
Appreciate the help.
I'll leave some feedback as soon as I test it.
Thank you

**Comment by soonick** [comment]
Hello,

I was apart from this issue for a while, and I am trying to remember how this feature was going to work. If someone can explain how this feature should work I can try to dedicate some time to implementing it.

Thanks,
Adrian

**Comment by argyrism** [comment]
Hello Adrian,

I have uploaded a working version with lists, unfortunately i do not 
have time now to get involved, if you manage to setup the database 
connections, you will be able to use that custom version, i think there 
are some instructions i have uploaded with in the past.

Regards,
Argyris

**Comment by soonick** [comment]
I am closing this issue since the creator can't get involved right now. If anybody else needs this feature a new ticket can be opened were we can discuss how this will work.
